src/plugins/kibana/public/settings/sections/status/index.js

```diff
@@ -5,6 +5,6 @@ define(function (require) {
     order: 3,
     name: 'status',
     display: 'Status',
-    url: '/status'
+    url: '../status'
```

Comment:
Yeah, this is a lot simpler but the other option is more consistent and a lot easier to maintain (though it costs a bit up front). I'll get this added.